railties/railties.gemspec

```diff
@@ -36,6 +36,8 @@ Gem::Specification.new do |s|
   s.add_dependency "rake", ">= 0.8.7"
   s.add_dependency "thor", ">= 0.18.1", "< 2.0"
   s.add_dependency "method_source"
+  # FIXME: Drop maximum version requirement when minimum Ruby requirement is 2.5
+  s.add_dependency "did_you_mean", "<= 1.1.2"
```

Comment:
`did_you_mea` gem is bundled since Ruby 2.3. Is this comment correct?
https://www.ruby-lang.org/en/news/2015/12/25/ruby-2-3-0-released/

Reply:
The comment is correct, did_you_mean versions after 1.1.2 [only has support for Ruby 2.5.0 and later](https://github.com/yuki24/did_you_mean/blob/master/CHANGELOG.md#v120)

Reply:
Understood. But, on the contrary, if using Ruby 2.5, can not use the new did_you_mean. I think it is not good to restrict it with libraries.